src/node/cli.ts

```diff
@@ -275,3 +283,54 @@ export const parse = (argv: string[]): Args => {
 
   return args
 }
+
+const defaultConfigFile = `
+auth: password
+bind-addr: 127.0.0.1:8080
+`.trimLeft()
+
+// readConfigFile reads the config file specified in the config flag
+// and loads it's configuration.
+//
+// Flags set on the CLI take priority.
+//
+// The config file can also be passed via $CODE_SERVER_CONFIG and defaults
+// to ~/.config/code-server/config.yaml.
+export async function readConfigFile(args: Args): Promise<Args> {
```

Comment:
It feels a bit odd that this also merges the existing args, should we name it something like `mergeConfigFile`? Or do something like `Object.assign(readConfigFile(args.config), args)`?

Reply:
Fixed.